### PR TITLE
Split (better) data into test and train subsets

### PR DESCRIPTION
**Pull request by ljdursi**
Here I use better training features (defined in config.json) and split the data into a training and test set. We want to train the model on the training set, then do testing (predictions, and confusion matrix) on the test set.

### CODE REVIEW

File: data/config.json [hunk 1]
```diff
@@ -1,42 +1,84 @@
...
+        "13": "data/train/chr13.vcf.gz",
+        "14": "data/train/chr14.vcf.gz",
+        "15": "data/train/chr15.vcf.gz",
+        "16": "data/train/chr16.vcf.gz",
+        "17": "data/train/chr17.vcf.gz",
+        "18": "data/train/chr18.vcf.gz",
+        "19": "data/train/chr19.vcf.gz",
+        "20": "data/train/chr20.vcf.gz",
+        "21": "data/train/chr21.vcf.gz",
+        "22": "data/train/chr21.vcf.gz"
```
Comment (Samarth-Agarwal1) [inline data/config.json]: Should this be chr22.vcf.gz instead of chr21.vcf.gz?

**Comment by ljdursi** [comment]
Also added a trivial test case

File: data/scripts/get_data.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+tabix -h s3://1000genomes/release/20130502/ALL.chr15.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz 15:28197036-28197038 15:28365617-28365619 15:36220034-36220036 15:45152370-45152372 15:48426483-48426485 > chr15.vcf
+tabix -h s3://1000genomes/release/20130502/ALL.chr16.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz 16:89730826-89730828 > chr16.vcf
+tabix -h s3://1000genomes/release/20130502/ALL.chr17.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz 17:40658532-40658534 17:41056244-41056246 17:48726131-48726133 17:53568883-53568885 17:62987150-62987152 > chr17.vcf
+tabix -h s3://1000genomes/release/20130502/ALL.chr18.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz 18:35277621-35277623 18:40488278-40488280 18:67578930-67578932 18:67867662-67867664 > chr18.vcf
+tabix -h s3://1000genomes/release/20130502/ALL.chr19.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz 19:4077095-4077097 > chr19.vcf
+tabix -h s3://1000genomes/release/20130502/ALL.chr20.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz 20:62159503-62159505 > chr20.vcf
```
Comment (Samarth-Agarwal1) [inline data/scripts/get_data.sh:22]: This line outputs to chr20.vcf and the line directly after this outputs to chr22.vcf. Should there have been a line that outputs to chr21.vcf?

File: src/id3_variants_training/ID3_Class.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import math
+import json
```
Comment (Samarth-Agarwal1) [inline src/id3_variants_training/ID3_Class.py]: `import json` seems to be an unused import.

Comment (ljdursi) [reply src/id3_variants_training/ID3_Class.py]: Good catch!

File: data/scripts/get_data.sh (hunk 2 above)
Comment (ljdursi) [reply data/scripts/get_data.sh:22]: We weren't using 21, but it turns out we don't use 22 or 7 either, so I got rid of those lines.

File: data/config.json (hunk 1 above)
Comment (ljdursi) [reply data/config.json]: I got rid of 21, 22, and 7 - we don't use those chromosomes.

**Review by Samarth-Agarwal1 — APPROVED** [review]
